cypress/integration/smokeTests/sharedTests/auth-tests.ts

```diff
@@ -182,17 +182,23 @@ function testTool(registry: string, repo: string, name: string) {
   // disable test until hiding versions for Tools are working on dev
 
   describe('Hide and un-hide a tool version', () => {
-    registerQuayTool(repo, name);
-    it('hide a version', () => {
-      goToTab('Versions');
+    function toggleHidden() {
       cy.contains('button', 'Actions').should('be.visible');
       cy.contains('td', 'Actions').first().click();
       cy.contains('button', 'Edit').click();
       cy.contains('div', 'Hidden:').within(() => {
         cy.get('[name=checkbox]').click();
       });
       cy.contains('button', 'Save Changes').click();
+    }
+    registerQuayTool(repo, name);
+    it('hide a version', () => {
+      goToTab('Versions');
+      toggleHidden();
```

Comment:
Since it's easy, we could add a test for the precondition (the hidden check shouldn't exist) before the first toggle (here and in the workflow version below), thus foreclosing some possible avenues of later confusion if things go wrong...

Reply:
Looked at it briefly, it's not as trivial as it seems to do this. I'd like to get this in to automate a little bit more of the 1.13 release, so going to punt.